.github/workflows/publish.yml

```diff
@@ -45,11 +45,11 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Set up JDK 8
+      - name: Set up JDK 11
         uses: actions/setup-java@v3
         with:
           distribution: temurin
-          java-version: 8
+          java-version: 11
```

Comment:
should we change the java version for the jar publish too?

Reply:
It should be safe due to https://github.com/apache/incubator-pekko-http/pull/187 but lets do this in a separate PR because I want to validate on my machine that the bytecode is exactly the same.